content/webapp/pages/season.tsx

```diff
@@ -13,82 +13,87 @@ import { contentLd } from '@weco/common/utils/json-ld';
-      contributorsTitle: season.contributorsTitle,
-      promo: season.promo,
-      body: season.body,
-      standfirst: season.standfirst,
-      promoImage: season.promoImage,
-      promoText: season.promoText,
-      image: season.image,
-      squareImage: season.squareImage,
-      widescreenImage: season.widescreenImage,
-      labels: season.labels,
-      metadataDescription: season.metadataDescription,
-    };
+  const ContentTypeInfo = season.standfirst && (
+    <PageHeaderStandfirst html={season.standfirst} />
+  );
+  const FeaturedMedia = getFeaturedMedia(genericFields);
+  const Header = (
+    <PageHeader
+      breadcrumbs={{ items: [] }}
+      labels={{ labels: season.labels }}
+      title={season.title}
+      ContentTypeInfo={ContentTypeInfo}
+      Background={<HeaderBackground hasWobblyEdge={true} />}
+      FeaturedMedia={FeaturedMedia}
+      HeroPicture={null}
+    />
+  );
 
-    const ContentTypeInfo = season.standfirst && (
-      <PageHeaderStandfirst html={season.standfirst} />
-    );
-    const FeaturedMedia = getFeaturedMedia(genericFields);
-    const Header = (
-      <PageHeader
-        breadcrumbs={{ items: [] }}
-        labels={{ labels: season.labels }}
-        title={season.title}
-        ContentTypeInfo={ContentTypeInfo}
-        Background={<HeaderBackground hasWobblyEdge={true} />}
-        FeaturedMedia={FeaturedMedia}
-        HeroPicture={null}
+  return (
+    <PageLayout
+      title={season.title}
+      description={season.metadataDescription || season.promoText || ''}
+      url={{ pathname: `/seasons/${season.id}` }}
+      jsonLd={contentLd(season)}
+      siteSection={'whats-on'}
+      openGraphType={'website'}
+      imageUrl={season.image && convertImageUri(season.image.contentUrl, 800)}
+      imageAltText={season.image && season.image.alt}
+    >
+      <ContentPage
+        id={season.id}
+        Header={Header}
+        Body={<Body body={season.body} pageId={season.id} />}
+      ></ContentPage>
+      <CardGrid
+        items={[...articles, ...books, ...events, ...exhibitions]}
+        itemsPerRow={4}
       />
-    );
+    </PageLayout>
+  );
+};
 
-    return (
-      <PageLayout
-        title={season.title}
-        description={season.metadataDescription || season.promoText || ''}
-        url={{ pathname: `/seasons/${season.id}` }}
-        jsonLd={contentLd(season)}
-        siteSection={'whats-on'}
-        openGraphType={'website'}
-        imageUrl={season.image && convertImageUri(season.image.contentUrl, 800)}
-        imageAltText={season.image && season.image.alt}
-      >
-        <ContentPage
-          id={season.id}
-          Header={Header}
-          Body={<Body body={season.body} pageId={season.id} />}
-        ></ContentPage>
-        <CardGrid
-          items={[...articles, ...books, ...events, ...exhibitions]}
-          itemsPerRow={4}
-        />
-      </PageLayout>
-    );
+SeasonPage.getInitialProps = async (
```

Comment:
curious any reason we use getinitialProps rather than getServersideprops?

Reply:
No, it was a bit of copy and paste just to get things going - it's my intention to change this to ServerSideProps as we continue working on it